### ISSUE TITLE
Warning about deprecate `set-output` command still visible in `v1.0.8` 

### ISSUE THREAD

**Issue by kimlep01** [joerick/pr-labels-action#12]
Hi, I updated my workflow to use `v1.0.8` and I'm still seeing two warnings about `set-output`. I created a new issue because no one reacted to my earlier comment.

My repo:
```
Run joerick/pr-labels-action@v1.0.8

Found label  do-not-merge 
  Setting env var for remaining steps: GITHUB_PR_LABEL_DO_NOT_MERGE=1

Action output:
labels: " do-not-merge "
labels-object: {"do-not-merge":true}
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Latest PR check in this repo https://github.com/joerick/pr-labels-action/actions/runs/4469843040/jobs/7852529676: 
```
Run ./
Not a pull request
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/[2](https://github.com/joerick/pr-labels-action/actions/runs/4469843040/jobs/7852529676#step:3:3)022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

This will start failing soon on June 1st.

_Originally posted by @kimlep01 in https://github.com/joerick/pr-labels-action/issues/9#issuecomment-1490024058_

**Comment by alexciesielski** [comment]
I opened [this PR](https://github.com/joerick/pr-labels-action/pull/13) which should fix the deprecation warning.

**Comment by joerick** [comment]
Fixed in v1.0.9 thanks to @alexciesielski !
